src/DynamoCoreWpf/ViewModels/Core/DynamoViewModel.cs

```diff
@@ -583,6 +604,11 @@ public static DynamoViewModel Start(StartConfiguration startConfiguration = new
 
         protected DynamoViewModel(StartConfiguration startConfiguration)
         {
+            // These options are currently fixed. Eventually we should make them dynamic I guess.
+            defaultPythonEngineOptions.Add(new NameValuePairViewModel<string>(string.Empty, WpfResources.DefaultPythonEngineNone));
+            defaultPythonEngineOptions.Add(new NameValuePairViewModel<string>("IronPython2", "IronPython2"));
+            defaultPythonEngineOptions.Add(new NameValuePairViewModel<string>("CPython3", "CPython3"));
```

Comment:
Sry to get to it late, this is the reason why I mentioned earlier if we want to just use the enums we already exposing as `PythonEngineVersion`, we just need to add a default entry

Reply:
I guess what I mean is instead of defining a fix collection but loop through the enums and construct it to reduce defined strings. Not sure if you have thought about it already

Reply:
The thing is that while `IronPython2` and `CPython3` are engine names and can go as they are, `Unspecified` is the absence of an engine and should be translated. Probably in different ways according to the context.
Also, not sure how permanent our enum is going to live forever. If we are going to open up the possibility for new engines, we'll need a solution based on names/strings.